addons/website_sale/tests/test_website_sale_pricelist.py

```diff
@@ -229,7 +227,8 @@ def test_pricelist_combination(self):
                 'product_uom': product.uom_id.id,
                 'price_unit': product.list_price,
                 'tax_id': False,
-            })]
+            })],
+            'pricelist_id': website_pricelist.id,
```

Comment:
If you're inspired, it could make sense to move the website pricelist computation to an override of `_compute_pricelist_id` considering the `website_id` of the sale order :thinking: